src/core_modules/capture-core-utils/RulesEngine/rulesEffectsProcessor/rulesEffectsProcessor.js

```diff
@@ -259,22 +260,18 @@ export default function getRulesEffectsProcessor(
         effects: Array<ProgramRuleEffect>,
         processType: $Values<typeof processTypes>,
         dataElements: ?DataElements,
-        trackedEntityAttributes: ?TrackedEntityAttributes) {
+        trackedEntityAttributes: ?TrackedEntityAttributes): ?Array<OutputEffect> {
```

Comment:
to annotate this I had to switch the filter and the map functions bellow.

Reply:
What if we first filter for `mapActionsToProcessor[action]` and then also filter in the end for truth values?

This way we can avoid the pain of `FlowFixMe` and also have annotated the return of the function `mapProcessTypeToIdentifierName`

Reply:
If that for some reason helps flow, I guess that's ok.